package.json

```diff
@@ -129,7 +130,6 @@
     "commander": "^2.18.0",
     "debug": "^4.0.1",
     "get-stdin": "^6.0.0",
-    "https-proxy-agent": "^2.2.1",
```

Comment:
I think you just removed this dependency by accident

Reply:
It's needed - the `@types` go in dev dependencies and will only occur *in this repo*. Not in a consumer of danger